cmd/thanos/tools_bucket.go

```diff
@@ -153,6 +155,13 @@ type bucketMarkBlockConfig struct {
 	blockIDs []string
 }
 
+type bucketDeleteConfig struct {
```

Comment:
I am wondering if we need to rename it because `delete` command might be ambiguous, whether it deletes series or deletes the whole block.
Maybe we can rename it to `delete-series` or `create-tombstone`, as we may need `undelete-series` or `remove-tombstone` in the future...